packages/amplify_core_plugin_interface/pubspec.yaml

```diff
@@ -13,6 +13,8 @@ dependencies:
   amplify_storage_plugin_interface: ^0.0.1-dev.2
   amplify_auth_plugin_interface: ^0.0.1-dev.2
   amplify_analytics_plugin_interface: ^0.0.1-dev.2
+  amplify_datastore_plugin_interface:
+    path: ../amplify_datastore_plugin_interface
```

Comment:
I think it could be worth rebasing on master and using melos so that we don't need a mix of local paths and pub.dev versions.

Reply:
Discussed offline, this will be captured in next PR from @haverchuck